tests/acceptance/api_test.go

```diff
@@ -1559,6 +1559,17 @@ func TestApi(t *testing.T) {
 			assert.Equal(t, 2, len(results), "only two leases should be returned")
 		})
 
+		t.Run("When there is a principal ID and an Account ID parameter", func(t *testing.T) {
+			resp := apiRequest(t, &apiRequestInput{
+				method: "GET",
+				url:    apiURL + "/leases?principalId=" + principalIDOne + "&accountId=" + accountIDOne,
+				json:   nil,
+			})
+
+			results := parseResponseArrayJSON(t, resp)
+			assert.Equal(t, 1, len(results), "only one lease should be returned")
```

Comment:
do we have test coverage for empty results? Might be a good idea...

Reply:
It was a single object but is now an array of one. Is this really a valid test that asserts same-as-before-the-break functionality?

Reply:
Can we add a test to verify an empty array is returned when no record found?